repos/system_upgrade/el7toel8/actors/storagescanner/libraries/library.py

```diff
@@ -79,22 +80,46 @@ def _get_fstab_info(fstab_path):
     ''' Collect storage info from /etc/fstab file '''
     if _is_file_readable(fstab_path):
         with open(fstab_path, 'r') as fstab:
-            for entry in fstab:
+            keys = ['fs_spec', 'fs_file', 'fs_vfstype', 'fs_mntops', 'fs_freq', 'fs_passno']
+            for line, entry in enumerate(fstab, 1):
                 if entry.startswith('#'):
                     continue
 
+                if '#' in entry:
```

Comment:
This will generate a lot of false positives. Either remove this warning and check completely, or check that '#' is the first character of a word.